src/main/java/frc/robot/subsystems/gripper/GripperConstants.java

```diff
@@ -36,5 +55,23 @@ public static void initConstants() {
                 KD.initDefault(0);
                 break;
         }
+
+        MOTOR_CONFIGURATION
```

Comment:
Missing Motion Magic Expo configs

Reply:
e67952c29b3e69f6de236053f62ee2cb0848fb2c

Reply:
Again, no need for additional constants, unless they need to be tunable, which is a reasonable use case here.

Reply:
fb1cace7c0294af728136a2cca0a13a221e35480